lms/static/js/writable_gradebook.js

```diff
@@ -312,59 +311,46 @@ $(document).ready(function() {
-            $modal.find('.grade-override-message').text(gettext('There are no student grades to adjust.'));
-            $modal.find('.grade-override-message').show();
-            $saveGradeOverrideButton.hide();
-        }
-        else {
-            $adjustedGradeHeader.text(isManualGrading ? 'Manual grade' : 'Adjusted grade');
-            $autoGradeHeader.text(isManualGrading ? 'Current grade' : 'Auto grade');
-
-            $manualGradeVisibilityWrapper.toggle(isManualGrading);
-            $saveGradeOverrideButton.attr('data-manual-grading', isManualGrading);
-            $manualGradeVisibilityWrapper.attr('data-visibility', gradesPublished);
-            $('input[name=grades-published]').prop('checked', gradesPublished);
-
-            $tableWrapper.attr('data-manual-grading', isManualGrading);
-            $tableWrapper.show();
-            $modal.find('#modal-table-empty-message').hide();
-            $saveGradeOverrideButton.show().prop('disabled', true);
-            modalDataTable.$('tr').each(function(){
-                $(this).attr('data-block-id', blockID);
-                var $adjustedGradePlaceholder = $(this).find('td.user-adjusted-grade');
-                var $autoGradePlaceholder = $(this).find('td.user-auto-grade');
-                var username = $autoGradePlaceholder.attr('data-username');
-
-                if (username in userAutoGrades) {
-                    $autoGradePlaceholder.text(userAutoGrades[username]);
-                    var autoEarnedGrade = userAutoGrades[username].split('/')[0],
-                        autoPossibleGrade = userAutoGrades[username].split('/')[1];
-                    $adjustedGradePlaceholder.attr('data-score-earned', autoEarnedGrade);
-                    $adjustedGradePlaceholder.attr('data-score-possible', autoPossibleGrade);
-                    $autoGradePlaceholder.attr('data-sort', autoEarnedGrade);
-
-                    if (username in userAdjustedGrades) {
-                        var adjustedGrade = userAdjustedGrades[username].split('/')[0];
-                        $adjustedGradePlaceholder.attr('data-score-earned', adjustedGrade);
-                        $adjustedGradePlaceholder.attr('data-sort', adjustedGrade);
-                        $adjustedGradePlaceholder.addClass('has-adjusted-score');
-                    }
-                    else if (isManualGrading) {
-                        $adjustedGradePlaceholder.attr('data-sort', autoEarnedGrade);
-                    }
-                    else {
-                        $adjustedGradePlaceholder.attr('data-sort', autoEarnedGrade);
-                    }
-                    $adjustedGradePlaceholder.find('input').val($adjustedGradePlaceholder.attr('data-score-earned'));
-                    $adjustedGradePlaceholder.find('span').text($adjustedGradePlaceholder.attr('data-score-possible'));
+        $adjustedGradeHeader.text(isManualGrading ? 'Manual grade' : 'Adjusted grade');
```

Comment:
Should we change 'Adjusted Grade' to something else?  "Adjusted" feel kind of confusing, especially if this is the first time I've gone in to modify grades for this course or subsection.

Reply:
Hmm.. What are you thinking then? Maybe just "Grade"?

Reply:
Yeah, something more generic like that.

Reply:
Done